uid2_client/encryption.py

```diff
@@ -147,6 +149,86 @@ def _decrypt_token_v3(token_bytes, keys, now):
     return DecryptedToken(id_str, established, site_id, site_key.site_id)
 
 
+def _encrypt_token(uid2, identity_scope, master_key, site_key, site_id, now, token_expiry, ad_token_version):
+    site_payload = bytearray(128)
+    #Site id
+    site_payload[0:4] = int.to_bytes(site_id, byteorder='big', length=4)
+    site_payload[4:12] = int.to_bytes(0, byteorder='big', length=8)
```

Comment:
can we add comments for each field, similar to https://github.com/IABTechLab/uid2-client-java/blob/master/src/main/java/com/uid2/client/Uid2TokenGenerator.java#L84